data/base-picking.yaml

```diff
@@ -103,3 +103,20 @@ picking:
     slim copper: 2706
     quality copper: 10824
     slim diamondique: 27060
+    # Riverhaven
+    ordinary: 100
```

Comment:
I'm not quite sure what it'll do if the same key appears multiple times. I suspect it'll only store the last one. Can you double check how this behaves?

Reply:
With this base-picking.yaml I went to crossing, bought an empty lockpick ring and attempted to pick a box. It refilled the ring with 250 kronar stout lockpicks in crossing that are also in the haven list for 200. Is that the right way to test it?

Reply:
I'm not concerned that it will buy the wrong lockpicks. Rather, I'm concerned that it will withdraw the wrong amount of coin.

I confirmed that if the same key appears, the last value is used. That means that for the Crossing, only 200 copper will be withdrawn per `stout` lockpick (instead of 250). I suggest depositing everything, then double checking how much coin gets withdrawn.

Reply:
Would it make more sense to just defer to the crossing cost instead? Since its higher, you will be able to afford both. Maybe leave a comment saying that it exists in riverhaven and the cost is 100 incase the schema ever gets modified

Reply:
That was my thinking too